lab_1/main.py

```diff
@@ -49,7 +79,11 @@ def get_top_n_words(freq_dict: dict, top_n: int) -> list:
     top_n = 1
     --> ['happy']
     """
-    pass
+    max_list = list(zip(d.values(),d.keys()))
+    max_list.sort()
+    max_token = list(max_list[-1])
+    (max_token).pop(0)
+
```

Comment:
there probably should be one more line written, don t you think so? (return) (x2)